pkg/config/draftconfig.go

```diff
@@ -6,6 +6,7 @@ import (
 
 //TODO: remove Name Overrides since we don't need them anymore
 type DraftConfig struct {
+	DisplayName      string              `yaml:"displayName"`
 	NameOverrides    []FileNameOverride  `yaml:"nameOverrides"`
```

Comment:
would removing NameOverrides be a larger change at this point than just deleting it from the struct definition?